test/test_HPE3ParClient_volume.py

```diff
@@ -17,7 +17,7 @@
 import time
 import unittest
 from testconfig import config
-
+import mock
```

Comment:
SDK tests are to be written using flask and not mock. For now, please remove the test file from this PR. 
Later tests can be written for this.